build/Install-WindowsSdkISO.ps1

```diff
@@ -0,0 +1,334 @@
+[CmdletBinding()]
+param([Parameter(Mandatory=$true, Position=0)]
+      [string]$buildNumber)
+
+# Ensure the error action preference is set to the default for PowerShell3, 'Stop'
+$ErrorActionPreference = 'Stop'
+
+# Constants
+$WindowsSDKOptions = @("OptionId.UWPCpp", "OptionId.UWPManaged", "OptionId.DesktopCPPx86", "OptionId.DesktopCPPx64")
+$WindowsSDKRegPath = "HKLM:\Software\Microsoft\Windows Kits\Installed Roots"
+$WindowsSDKRegRootKey = "KitsRoot10"
+$WindowsSDKVersion = "10.0.$buildNumber.0"
+$WindowsSDKInstalledRegPath = "$WindowsSDKRegPath\$WindowsSDKVersion\Installed Options"
+$StrongNameRegPath = "HKLM:\SOFTWARE\Microsoft\StrongName\Verification"
+$PublicKeyTokens = @("31bf3856ad364e35")
+
+if ($buildNumber -notmatch "^\d{5,}$")
+{
+    Write-Host "ERROR: '$buildNumber' doesn't look like a windows build number"
```

Comment:
Thanks for taking a look at all this. I'm just reusing the same script that MUX is using and I don't want to make it diverge too much from that at this point, especially since there are really no functional concerns here. I'll keep this as it is.

Reply:
Yeah, I saw this was just copied over. If this lives more than just temporarily in this repo, though, I feel like it should be cleaned up, especially since some of these things seem like bugs. That can be a future PR I guess.